lib/t/calls.ex

```diff
@@ -64,8 +65,8 @@ defmodule T.Calls do
     |> Repo.exists?()
   end
 
-  defp live_mode?() do
-    T.Feeds.is_now_live_mode()
+  defp in_live_mode?(caller_id, called_id, reference) do
+    Feeds.live_now?(caller_id, reference) and Feeds.live_now?(called_id, reference)
```

Comment:
potential #TODO - do not allow calls for blocked / reported
i.e. during LIVE:
A invites B
B calls A
B harrases A, A ends the call & reports B
B still can call A

Reply:
moved to https://github.com/getsince/test3/issues/447